internal/controller/components/kueue/kueue.go

```diff
@@ -32,10 +42,11 @@ func (s *componentHandler) GetName() string {
 }
 
 func (s *componentHandler) GetManagementState(dsc *dscv1.DataScienceCluster) operatorv1.ManagementState {
-	if dsc.Spec.Components.Kueue.ManagementState == operatorv1.Managed {
+	state := dsc.Spec.Components.Kueue.ManagementState
+	if state == operatorv1.Unmanaged || state == operatorv1.Managed {
```

Comment:
this will return `Managed` for both `Managed` and `Unmanaged` case?

Reply:
This is temporary, as today the whole component lifecycle logic is based on `Managed/Removed` so we would need to also introduce a proper handling for the `Unmanaged` case